common/defs-cmds.go

```diff
@@ -79,9 +81,19 @@ const (
 	ScheduleDescribeDefinition = "Get Schedule configuration and current state."
 	ScheduleDeleteDefinition   = "Deletes a Schedule."
 	ScheduleListDefinition     = "Lists Schedules."
+
+	// Update build id subcommand definitions
+	AddNewDefaultBuildIDDefinition         = "Add a new default (incompatible) build ID to the Task Queue version sets."
+	AddNewDefaultBuildIDDefinitionUsage    = "Creates a new build id set which will become the new overall default for the queue with the provided build id as its only member. This new set is incompatible with all previous sets/versions."
```

Comment:
nit:

```suggestion
	AddNewDefaultBuildIDDefinitionUsage    = "Creates a new build ID set which will become the new overall default for the queue with the provided build ID as its only member. This new set is incompatible with all previous sets/versions."
```

I would also see how this renders and consider adding line breaks.

Reply:
Any semi-competent CLI flags library should be able to do this automatically (which this one maybe barely qualifies as lol). It looks like the one we're using has some kind of support for that: https://github.com/urfave/cli/pull/1119/files#diff-4d3f36d62cad94424544bb84fc93b33b73ffe3491a4ddca81a7f825539afc19cR1161-R1169

So we might consider doing that in another PR rather than manually inserting line breaks in a few places.